numpy/core/tests/test_numeric.py

```diff
@@ -585,6 +585,9 @@ def test_errobj_noerrmask(self):
 
 
 class TestFloatExceptions:
+    @pytest.mark.xfail(sys.platform == 'cygwin',
+                       reason='FPE support incomplete',
+                       strict=True)
```

Comment:
Could you elucidate what you mean by "FPE support incomplete"?  What FPE APIs are unavailable on Cygwin?  I seem to recall more recent versions of Cygwin have pretty good support for floating point exceptions.

Reply:
I know this test failed a few months ago; that may have been one of the things that got fixed in the last six months or so.  Unfortunately "this test involves FPEs and fails" is about the sum of my knowledge on the subject.

Reply:
This may only apply to the test documented a few functions down; since this is applied to an assert function rather than a test function, it may not do anything useful.  It may be a good idea to drop this mark and see what happens.

Reply:
I dropped this mark and have noticed no new failures.